src/main/webapp/src/app/ranking/ranking-list/ranking-list.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, SimpleChanges } from '@angular/core';
```

Comment:
I am not sure what SImpleChanges does and if it was used anywhere in this file, could you pls put a comment or if it's not used maybe delete it